src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -89,7 +123,12 @@ object TezosTypes {
     final case class Contracts(
       storage: Micheline,
       code: Micheline
-    )
+    ) {
+      def flatMap(storageFunction: String => Option[String], codeFunction: String => Option[String]): Option[Contracts] = for {
```

Comment:
Although the functionality if clear, could you please still document this function?